Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -53,7 +53,7 @@ def __init__(
         risk_score_threshold: int = 0,
         tags: list | None = None,
         tlp_color: str | None = None,
-        performance: bool = False,
+        performance: bool = True,
```

Comment:
Is this an intentional change? `performance` should be enabled by default, so good catch

Reply:
Yes, I changed it intentionally.